lib/compress/zstd_compress.c

```diff
@@ -1251,19 +1251,23 @@ void ZSTD_seqToCodes(const seqStore_t* seqStorePtr)
 }
 
 MEM_STATIC symbolEncodingType_e ZSTD_selectEncodingType(FSE_repeat* repeatMode,
-        size_t const mostFrequent, size_t nbSeq, U32 defaultNormLog)
+        size_t const mostFrequent, size_t nbSeq, U32 defaultNormLog, U32 basic)
```

Comment:
Is `basic` a good name ?
I find it a bit difficult to associate what it is about.
Of course, since we have been talking about it, I can follow. But it's easy to lose this background context.